valid-url/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```

Comment:
I've updated this, but I'm not sure why it needs doing (all examples I looked at have it set to false)?